jsonb/src/main/java/io/avaje/jsonb/stream/Recycle.java

```diff
@@ -1,38 +1,58 @@
 package io.avaje.jsonb.stream;
 
-import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
 final class Recycle {
+  private Recycle() {}
+
+  private static boolean jvmRecycle;
+  private static ThreadLocal<JParser> read;
+  private static ThreadLocal<JGenerator> managed;
+
+  static {
+    if (Float.parseFloat(System.getProperty("java.specification.version")) >= 19
+        && !Boolean.getBoolean("jsonb.UseTLBuffers")) {
```

Comment:
I think I'll adjust this from "jsonb.UseTLBuffers" to "jsonb.useTLBuffers" with the lower case u for useTLBuffers